lib/active_merchant/billing/gateways/paypal_complete.rb

```diff
@@ -106,7 +106,18 @@ def add_payment_source(post, vault_id, options)
         post[:payment_source] ||= {}
         payment_source = options[:payment_type] == "paypal_account" ? :paypal : :card
         post[:payment_source][payment_source] = {
-          vault_id: vault_id
+          vault_id: vault_id,
+          experience_context: {
+            payment_method_preference: "IMMEDIATE_PAYMENT_REQUIRED",
+            brand_name: "Maxio",
```

Comment:
Is it ok to hard-code this value?

Reply:
I think it's ok unless we are planning to change company name anytime soon 😅